Dockerfile

```diff
@@ -1,10 +1,10 @@
 FROM tarantool/tarantool
 
-RUN mkdir -p /etc/tarantool/client-tests
+RUN mkdir -p /opt/tarantool/work_dir
```

Comment:
For persisting data and logging.

> On 23 Jun 2016, at 10:28 , roman-kozachenko <notifications@github.com> wrote:
> 
> In Dockerfile <https://github.com/aensidhe/tarantool-csharp/pull/30#discussion_r68187234>:
> 
> > @@ -1,10 +1,10 @@
> >  FROM tarantool/tarantool
> >  
> > -RUN mkdir -p /etc/tarantool/client-tests
> > +RUN mkdir -p /opt/tarantool/work_dir
> What work_dir is used for?
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub <https://github.com/aensidhe/tarantool-csharp/pull/30/files/d15deea9997a8e59d0e13d1fc59e2feae938690b#r68187234>, or mute the thread <https://github.com/notifications/unsubscribe/ADw9-ySZUfSAPG_KVS2sX8Lx1c_D1f0-ks5qOjWfgaJpZM4I5n7g>.
>